### PR TITLE
ocrvs-4485: On work queue users will only see corresponding location declarations

### PR DESCRIPTION
**Pull request by yeasinhossain-dsi**

### CODE REVIEW

**Review by Zangetsu101 — COMMENTED** [review]
@yeasinhossain-dsi Correct me if I'm wrong, only registrars will be able to search for declarations  from other offices after this right? And the workqueues will be how they were before by only showing the declarations of the user's office.

@euanmillar I think we wanted the RegistrationAgent to be able to search for declarations of other offices as well right?

**Comment by euanmillar** [comment]
> RegistrationAgent to be able to search for declarations of other offices as

Yes I think you are right Tameem.  Please also allow reg agents

File: packages/client/src/views/SearchResult/SearchResult.tsx [hunk 1]
```diff
@@ -346,9 +346,14 @@ export class SearchResultView extends React.Component<
...
                             : '',
                         name: searchType === NAME_TEXT ? searchText : '',
-                        declarationLocationId: userDetails
-                          ? getUserLocation(userDetails).id
-                          : ''
+                        declarationLocationId:
+                          userDetails &&
+                          ![
+                            Roles.LOCAL_REGISTRAR,
+                            Roles.NATIONAL_REGISTRAR
```
Comment (euanmillar) [inline packages/client/src/views/SearchResult/SearchResult.tsx:353]: Add Roles.REGISTRATION_AGENT @yeasinhossain-dsi

Comment (yeasinhossain-dsi) [reply packages/client/src/views/SearchResult/SearchResult.tsx]: Added

**Review by euanmillar — APPROVED** [review]

**Pull request merged**